microprofile.jdt/org.eclipse.lsp4mp.jdt.core/src/main/java/org/eclipse/lsp4mp/jdt/core/MicroProfileCorePlugin.java

```diff
@@ -57,6 +62,10 @@ public static MicroProfileCorePlugin getDefault() {
 		return plugin;
 	}
 
+	public ILog getLogger() {
+		return Platform.getLog(context.getBundle());
```

Comment:
Please don't use ILog in this PR. I think it should b ebetter to do that in another PR by replacing all LOGGER with ILog. We need to dicuss more if it is better to do that.